examples/cities.rs

```diff
@@ -0,0 +1,82 @@
+//! This example program scans all OSM nodes and extracts list of cities with
+//! name and population in JSON format.
+
+use flatdata::Archive;
+use osmflat::{FileResourceStorage, Osm};
+use serde::Serialize;
+
+use std::env;
+use std::error::Error;
+use std::io;
+
+#[derive(Debug, Default, Serialize)]
+struct City {
+    name: String,
+    population: usize,
+}
+
+/// Strings in an osmflat archive are stored in a single bytes blob separated by
+/// `0` character. Given an index, this function extracts a utf8 string starting
+/// at this index from the blob.
+fn get_str(strings: &[u8], start: usize) -> &str {
```

Comment:
I think we really need to move this to a helper in `osmflat`

Reply:
Makes sense. I will try to solve #25 first, and then merge this example.

Reply:
or even in flatdata-rs

Reply:
Don't we need a string block type in flatdata for that?